packages/spanner/system-test/spanner.js

```diff
@@ -2716,7 +2716,7 @@ var spanner = new Spanner(env);
           transaction.run('SELECT * FROM TxnTable', function(err, rows) {
             assert.ifError(err);
 
-            assert.strictEqual(rows.length, 2);
+            assert.strictEqual(rows.length, 1);
```

Comment:
Still 3 consistently. Here are the rows being returned (`console.log(rows)` added on the line above this change):

```
[ [ { name: 'Key', value: 'k0' },
    { name: 'StringValue', value: 'v0' },
    { name: 'NumberValue', value: null } ],
  [ { name: 'Key', value: 'k1' },
    { name: 'StringValue', value: 'v1' },
    { name: 'NumberValue', value: null } ],
  [ { name: 'Key', value: 'k2' },
    { name: 'StringValue', value: 'v2' },
    { name: 'NumberValue', value: null } ] ]
```